lib/tasks/release.rake

```diff
@@ -185,6 +185,10 @@ namespace :release do
     end
 
     update_gems = ENV["UPDATE_GEMS"].to_s.split(" ")
+    if ENV["UPDATE_PLUGINS"]
+      require "vmdb/plugins"
+      update_gems += Vmdb::Plugins.paths.values.map { |p| p.match(%r{/gems/(.+)-\h+$}).captures.first }
```

Comment:
This regex is checking the on-disk path (e.g. `/Users/jfrey/.gem/ruby/3.0.6/bundler/gems/manageiq-providers-workflows-a7c24c7cdd32`), then pulling out the gem/repo name (e.g. `manageiq-providers-workflows`)

I'm not sure if there's a better way?  I guess I could also take the engine name and underscorize it, but I think there's some inconsistencies like manageiq-consumption provides ManageIQ::Showback[^1]

[^1]: https://github.com/ManageIQ/manageiq-consumption/tree/561f74d85a026dc82b3b8bfa4a53f25edfc609bb/lib/manageiq